cmd/server/tools.go

```diff
@@ -18,6 +18,7 @@
 // OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 // THE SOFTWARE.
 
+//go:build tools
```

Comment:
nit: is this generated?

Reply:
Yes auto-generated by editor when running `make bins`